src/slack/slack.js

```diff
@@ -1159,9 +1159,15 @@ export class Slack {
 
   async getOrderAndItemsFromDb(orderId) {
     return await knex.transaction(async (trx) => {
+      const fields = ['id', 'isCompany', 'user', 'office', 'isUrgent', 'isHome']
+
+      if (this.variant !== 'wincent') {
```

Comment:
Do you think this will work for the test variant? I think I only added the field to Vacuumlabs, so it might be better to check here for Vacuumlabs instead 🤔

Reply:
There is a migration in master adding the column to 'testOrder' table - is that not for test variant?

Reply:
Oh, you are right, no problem then :)